MaxText/configs/base.yml

```diff
@@ -186,3 +186,11 @@ stack_trace_interval_seconds: 600  # Stack trace collection frequency in seconds
 
 # Use iota operator in Embed
 use_iota_embed: False
+
+# Xaot options
+xaot_save_name: "" # Name of saved serialized compiled train_step, e.g. compiled_train_v5e-256.pickle
+num_xaot_devices: 0
+save_xaot: False
+load_xaot: False
+topology: 'v4-8'
+topology_num_slices: 1
```

Comment:
xpk knows how to do this.

Reply:
Created a static map in hardware_topology. Will create a followup PR so XPK and XAOT use the same map